docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+Note that this bead-to-bead distance typically already has the bead diameter subtracted, therefore it reflects the surface-to-surface distance.
+We can plot what this curve looks like by invoking `.plot()` on it::
+
+    distance_calibration.plot()
+
+.. image:: figures/piezotracking/mirror_calibration.png
+
+We can also inspect the residual, to determine how well the calibration model describes the data::
+
+    distance_calibration.plot_residual()
+
+.. image:: figures/piezotracking/mirror_calibration_residual.png
+
+Baseline correction
+-------------------
+
+Even without a tether, the measured force can be non-zero when the beads are close together.
+The next step is to characterize this baseline force so that we can subtract it from our force data.
+In principle, this step is optional, but it can improve the accuracy of your force-distance curves.
+We can quickly determine a polynomial baseline for both traps by invoking::
+
+    baseline_1x = lk.ForceBaseLine.polynomial_baseline(
+        no_tether_data['Trap position']['1X'], no_tether_data.force1x, degree=7, downsampling_factor=100
+    )
+    baseline_2x = lk.ForceBaseLine.polynomial_baseline(
+        no_tether_data['Trap position']['1X'], no_tether_data.force2x, degree=7, downsampling_factor=100
+    )
+
+Similarly as before, we can plot these baselines to verify that they describe the data well::
+
+    baseline_1x.plot(label="baseline f1x")
+    baseline_2x.plot(label="baseline f2x")
+    plt.legend()
+
+.. image:: figures/piezotracking/baseline.png
+
+And the residuals::
+
+    baseline_1x.plot_residual(label="baseline f1x")
+    baseline_2x.plot_residual(label="baseline f2x")
+    plt.legend(loc='lower right')
+
+.. image:: figures/piezotracking/baseline_residual.png
+
+The residuals should be distributed around zero and should ideally look like noise.
+
+Calculating the piezo distance and corrected force
+--------------------------------------------------
+
+When a tether is present, it exerts a force on the beads.
+This force results in the displacement of the beads from the trap centers.
+Assuming only small excursions from the trap center, we can assume this relation to be linear (proportional to the trap stiffness :math:`\kappa`).
+Therefore, we can compute the bead displacement :math:`\delta x` directly from the force signal.
+
+.. math::
+
+    \delta x_1 = \frac{F_{1x}}{\kappa_{1x}}
+
+This means that the surface-to-surface distance can be computed by correcting the calibrated distance that was based on the trap, with the force signals and their respective stiffnesses as follows.
+
+.. math::
+
+    d_{piezo} = d_{no\_tether}(x_{trap1}) - \frac{F_{1x}}{\kappa_{1x}} + \frac{F_{2x}}{\kappa_{2x}}
+
+Here :math:`d_{piezo}` reflects the piezo distance, :math:`d_{no tether}` reflects the calibrated surface-to-surface distance without the tether with the trap position given by :math:`x_{trap1}`.
```

Comment:
Also remove the fragment referring to `x_trap1` if you decide to go with my suggestion above
```suggestion
Here :math:`d_{piezo}` reflects the piezo distance, :math:`d_{no_tether}` reflects the calibrated surface-to-surface distance without the tether with the trap position given by :math:`x_{trap1}`.
```